doc/user_guide/getting_started.rst

```diff
@@ -184,17 +184,20 @@ You are ready to use the toolbox. With *nox -l* you can list all available tasks
     $ nox -l
     Sessions defined in <PATH_TO_YOUR_PROJECT>/noxfile.py:
 
-    * fix -> Runs all automated fixes on the code base
-    - check -> Runs all available checks on the project
-    - lint -> Runs the linter on the project
-    - type-check -> Runs the type checker on the project
-    - unit-tests -> Runs all unit tests
-    - integration-tests -> Runs the all integration tests
-    - coverage -> Runs all tests (unit + integration) and reports the code coverage
+    * project:fix -> Runs all automated fixes on the code base
+    - project:check -> Runs all available checks on the project
+    - project:report -> Collects and generates metrics summary for the workspace
+    - test:unit -> Runs all unit tests
+    - test:integration -> Runs the all integration tests
+    - test:coverage -> Runs all tests (unit + integration) and reports the code coverage
+    - lint:code -> Runs the static code analyzer on the project
+    - lint:typing -> Runs the type checker on the project
+    - lint:security -> Runs the security linter on the project
+    - docs:multiversion -> Builds the multiversion project documentation
```

Comment:
Is this new or was forgotten in a previous PR

Reply:
Have been forgoten to add